docs/4.x/getting-started-part-1.md

```diff
@@ -69,6 +69,12 @@ The following tools aren't required for this guide, but you may find them useful
 - **[xhprof](https://github.com/facebook/xhprof)** If you'd like to profile your code and debug any inefficiencies.
 - **[python](https://www.python.org/)** If you want to use the log importer.
 
+**Important Note for Windows**
+
+It is possible to develop Matomo On Windows, but some things will not work as well. Specifically, Windows users
+coding with Vue may encounter problems with compiling Vue code. This is currently an unsolved problem, and we'd happily
+accept contributions that would help Windows users have an easier time contributing to Matomo.
+
```

Comment:
can you link an issue or provide more description of the problems on windows?

Reply:
there's no issue, but I can create one. There might be a lot of problems, I just gave up at the one that I couldn't fix :) . 

This problem being that the typescript compiler kept complaining about about relative imports (eg, `import ... from './ExampleComponent.vue`). The error was a bit vague, and from googling it seems like many people have this issue on windows w/ typescript.

Reply:
this was the error that stopped me:

Reply:
This is the issue that made me decide to stop working on it: https://github.com/vuejs/vue-cli/issues/2934

I'll put this in a new issue tomorrow.

Reply:
Created https://github.com/matomo-org/matomo/issues/18081